### DISCUSSION TITLE
can anyone provide me correct code to download deepmd GPU version

### DISCUSSION THREAD

**Discussion by experiment-23** [deepmodeling/deepmd-kit#3931] [Q&A]
(base) [user@login04 ~]$ conda create -n DeePMD deepmd-kit=*=gpu libdeepmd==*gpu lammps-dp cudatoolkit=11.3 horovod -c defaults
Collecting package metadata (current_repodata.json): / WARNING conda.models.version:get_matcher(544): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.7.1.*, but conda is ignoring the .* and treating it as 1.7.1
done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): - WARNING conda.models.version:get_matcher(544): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.8.0.*, but conda is ignoring the .* and treating it as 1.8.0
WARNING conda.models.version:get_matcher(544): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.6.0.*, but conda is ignoring the .* and treating it as 1.6.0
WARNING conda.models.version:get_matcher(544): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.9.0.*, but conda is ignoring the .* and treating it as 1.9.0
done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - deepmd-kit[build=gpu]
  - lammps-dp
  - libdeepmd==*gpu

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Comment by njzjz** [comment]
Please follow the command in the documentation https://docs.deepmodeling.com/projects/deepmd/en/v2.2.10/install/easy-install.html#install-with-conda
